manifests-maistra/1.0.0/maistra.v1.0.0.clusterserviceversion.yaml

```diff
@@ -83,7 +83,19 @@ metadata:
               }
             }
           }
-        }              
+        },
+        {
+          "apiVersion": "maistra.io/v1",
+          "kind": "ServiceMeshMemberRoll",
+          "metadata": {
+            "name": "default"
+          },
+          "spec": {
+            "members": [
+              "bookinfo"
```

Comment:
Does it make sense to include bookinfo here? Should we include some sort of a comment saying "include the proper namespaces" here?

Reply:
It's an example -- I figured including bookinfo is ok (or better than leaving the list empty). I think the idea is that it should be usable?

Reply:
If the user copies and paste this, and there's already a namespace called bookinfo in the cluster, then it might be a problem. Having it commented and also a comment `#Put the namespaces here blah` should work I think.